managed-token/program/src/instruction.rs

```diff
@@ -70,6 +70,30 @@ pub enum ManagedTokenInstruction {
     #[account(5, name = "freeze_authority")]
     #[account(6, name = "token_program", desc = "Token program")]
     CloseAccount,
+
+    #[account(0, name = "mint")]
+    #[account(1, writable, name = "account")]
+    #[account(2, signer, name = "owner")]
+    #[account(3, signer, name = "upstream_authority")]
+    #[account(4, name = "delegate")]
+    #[account(5, name = "freeze_authority")]
+    #[account(6, name = "token_program", desc = "Token program")]
+    Approve { amount: u64 },
+
+    #[account(0, name = "mint")]
+    #[account(1, writable, name = "account")]
+    #[account(2, signer, name = "owner")]
+    #[account(3, signer, name = "upstream_authority")]
+    #[account(4, name = "freeze_authority")]
+    #[account(5, name = "token_program", desc = "Token program")]
+    Revoke,
+
+    #[account(0, writable, name = "mint")]
+    #[account(1, signer, name = "mint_authority")]
+    #[account(2, signer, name = "freeze_authority")]
+    #[account(3, name = "upstream_authority")]
+    #[account(4, name = "token_program", desc = "Token program")]
+    Wrap,
```

Comment:
nit: the term `Wrap` typically implies a temporary move that can be undone (unwrapped), and this is giving full ownership of the mint to the program without any way to reverse.

How about something like `MigrateToProgramManagement` or `Migrate` to make it clearer that it's going away from the current manager?